### PR TITLE
hfkfo proposal for Boolean Operations on Polygons

### PR DESCRIPTION
**Pull request by hfkfo**
A good PR should include the following parts:

- [ ] Provide a clear title
- [ ] Provide meaningful descriptions in the PR and the commit logs
- [ ] Name your branch like ``<github-user-name>-hw#``, e.g., ``myname-hw2-attempt1``
- [ ] Merge from your specific branch (***not master***) to the `hw*` branch (***not master***, again)
- [ ] Exclude irrelevant files, such as binaries, text files, or dot files
- [ ] Make sure tests/CI pass

### CODE REVIEW

**Comment by yungyuc** [comment]
@hfkfo you should have created the PR against `project` branch.  I correct it for you.

**Review by yungyuc — CHANGES_REQUESTED** [review]
When requesting for a next review, please leave a global comment summarizing your change.

- [x] Change to use less than 80 characters in a line.  Long lines make it hard to annotate and discuss.
- [x] Add an entry in `project/README.rst` pointing to your proposal.

I will review again after you are ready.

**Comment by yungyuc** [comment]
Please add an entry in E3 too.  I can't find it.

**Comment by hfkfo** [comment]
I'm ready for next review

File: project/hfkfo/README.rst [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+ Problem to Solve
+******************
+**Boolean operations on polygons** involve combining two or more polygons using
+
+set operations like union, intersection, difference, and exclusive-or (XOR).
+
+.. image:: https://miro.medium.com/v2/resize:fit:1400/format:webp/1*-HFaUxQdqiibk27ovbT8QA.png
+
+Boolean operations on polygons are critical in various fields such as computer
+
```
Comment (yungyuc) [inline project/hfkfo/README.rst]: There should not be a blank line between lines in the same paragraph.

File: project/hfkfo/README.rst [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+****************************
+
+- Build : Make
+- Version control : git
+
+**********
+ Schedule
+**********
+
+* Week 1-2 (09/30): Setup the infrastructures and learn how to do CI in github.
```
Comment (yungyuc) [inline project/hfkfo/README.rst:75]: You will have a good start if basic testing and CI are ready by the end of the month.

File: project/hfkfo/README.rst [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+* Week 3-4 (10/14): Write C++ code to calculate Boolean Operations on Polygons. 
+* Week 5 (10/28): Write python code to visualize the result.
```
Comment (yungyuc) [inline project/hfkfo/README.rst]: Please specify how do you plan to visualize it.  Visualization is hard.

**Review by yungyuc — CHANGES_REQUESTED** [review]
- [x] [L81](https://github.com/yungyuc/nsdhw_24au/pull/46/files#diff-3b52f937846d878bd1b684a7b2eee3b818aa5a11cc61da36a7617071c7499545R81): Specify how to visualize.

**Comment by yungyuc** [comment]
Please also resolve merge conflict.

**Comment by hfkfo** [comment]
I'm ready for next review

**Pull request merged**